src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportFileUploader.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import {
+  notification,
+  Space,
+  Typography,
+} from "antd";
+import { DragUpload } from "../../../../components/files/DragUpload";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+import { SampleMetadataImportSteps } from "./SampleMetadataImportSteps";
+
+export function SampleMetadataImportFileUploader(props) {
+  const { Text, Title, Paragraph } = Typography
+  const options = {
+    multiple: false,
+    showUploadList: false,
+    accept: [".xls", ".xlsx", ".csv"],
+    action: setBaseUrl(`/projects/${props.projectId}/sample-metadata/upload/file`),
+    onChange(info) {
+      const { status } = info.file;
+      if (status !== 'uploading') {
+        console.log(info.file);
+      }
+      if (status === 'done') {
+        notification.success({
+          message: i18n("SampleMetadataImportFileUploader.success", info.file.name),
+        });
+      } else if (status === 'error') {
+        notification.error({
+          message: i18n("SampleMetadataImportFileUploader.error", info.file.name),
+        });
+      }
+    },
+  }
+  const [step, setStep] = useState(0);
+
+  return (
+    <>
+      <Space direction="vertical" size="large" style={{ width: `100%` }}>
+        <Typography>
```

Comment:
Why is this wrapped in Typography?  I don't think that does anything

Reply:
Please see 7385685aa01b9eec780f8980011cd02209f2e40b.